.github/workflows/check.yaml

```diff
@@ -22,16 +22,29 @@ jobs:
       tox-version: '<4'
 
   func:
```

Comment:
Please add concurrency, same as I did in another projects (example [charm-apt-mirror](https://github.com/canonical/charm-apt-mirror/pull/40). Thanks